components/Variant/Variant.tsx

```diff
@@ -0,0 +1 @@
+export type Variant = { variant?: "bright" | "dark" };
```

Comment:
If you want to reuse this you could call it VariantProps to indicate that it's not the actual variant which would be "bright" | "dark" but a props object.
Alternatively you could write it as a generic like
`export type WithVariantProps<AdditionalProps extends Record<string, any>> = {variant?: "bright" | "dark"} & AdditionalProps`
Also here you are using `type` whereas inside the components you're using `interface`.

No matter what you should store `"bright" | "dark"` as a type and probably call it Variant and rename this to VariantProps.

Reply:
btw is Variant a component? because it's under components.